RussianFilter/sections/specific.txt

```diff
@@ -8078,6 +8078,11 @@ www.ex.ua/*/index.css
 ||скачатьофис.рф/*.php?q=$script
 ||форум-мебельщиков.рф/banner/
 !
+! conversion.im: branding of the #mainContainer
+||conversion.im/storage/images/banners/
+conversion.im#%#//scriptlet("abort-on-property-read", "window.open")
+conversion.im#$##mainContainer { cursor: auto!important; }
```

Comment:
Exchange that scriptlet to this `conversion.im#%#//scriptlet("abort-current-inline-script", "$", "mainContainer")` and move to the right section `general_extensions` with comment above.
For CSS rule the same, but also re-write it to `conversion.im#$##mainContainer { cursor: auto!important; padding-top: 0!important; }`